packages/p2p/src/components/my-ads/toggle-ads.jsx

```diff
@@ -14,19 +14,21 @@ const ToggleAds = observer(() => {
     const isMounted = useIsMounted();
 
     const handleToggle = () => {
-        requestWS({
-            p2p_advertiser_update: 1,
-            is_listed: general_store.is_listed ? 0 : 1,
-        }).then(response => {
-            if (isMounted()) {
-                if (response.error) {
-                    setApiError(response.error.message);
-                } else {
-                    const { is_listed } = response.p2p_advertiser_update;
-                    general_store.setIsListed(is_listed === 1);
+        if (!general_store.is_barred) {
```

Comment:
Could we add a comment here explaining that a user's ads are automatically disabled/unlisted when they're barred from P2P?

Reply:
Done.